package/src/specs/RNCandle.nitro.ts

```diff
@@ -94,14 +313,14 @@ export type PresentationStyle = "sheet" | "fullScreen";
 export type State = "active" | "inactive";
 
 export type Details = {
-  state: State;
   username: string | undefined;
   legalName: string;
   accountOpened: string | undefined;
 };
 
 export type LinkedAccount = {
   serviceUserID: string;
+  state: State;
   details: Details | undefined;
   linkedAccountID: string;
   service: Service;
```

Comment:
#req switch this to your new approach for consistency, i.e. define `LinkedAccount`, `LinkedAccountDetails`, `ActiveLinkedAccountDetails`, `InactiveLinkedAccountDetails` just like the Swift models